pywinauto/controls/uia_controls.py

```diff
@@ -38,7 +38,7 @@
 from . import uiawrapper
 from . import win32_controls
 from . import common_controls
-from .. import findbestmatch
+from .. import findbestmatch, findwindows
```

Comment:
This import fails on Py2.7 on AppVeyor (restart shows it's reproducible). Please debug and fix.